apps/onboarding-functions/src/main/java/it/pagopa/selfcare/onboarding/service/CompletionServiceDefault.java

```diff
@@ -299,4 +306,13 @@ public void persistActivatedAt(Onboarding onboarding) {
                 .update("activatedAt = ?1 and updatedAt = ?2 ", now, now)
                 .where("_id", onboarding.getId());
     }
+
+    @Override
+    public Onboarding createAggregateOnboardingRequest(OnboardingAggregateOrchestratorInput onboardingAggregateOrchestratorInput) {
+        Onboarding onboardingToUpdate = onboardingMapper.mapToOnboarding(onboardingAggregateOrchestratorInput);
+        onboardingToUpdate.setWorkflowType(CONFIRMATION_AGGREGATE);
+        onboardingToUpdate.setStatus(OnboardingStatus.PENDING);
+        onboardingRepository.persistOrUpdate(onboardingToUpdate);
```

Comment:
pay attention that we should create a new onboarding record....it is important to set a new objectid for persisting a new object